include/z64.h

```diff
@@ -1111,13 +1111,13 @@ typedef struct SelectContext {
     /* 0x0230 */ s32 lockDown;
     /* 0x0234 */ s32 unk_234; // unused
     /* 0x0238 */ u8* staticSegment;
-} SelectContext; // size = 0x240
+} MapSelectState; // size = 0x240
 
 typedef struct {
     /* 0x0000 */ GameState state;
     /* 0x00A4 */ u8* staticSegment;
     /* 0x00A8 */ View view;
-} SampleContext; // size = 0x1D0
+} SampleState; // size = 0x1D0
```

Comment:
`SampleState` makes me think it could easily clash with something from audio as those docs continue. But maybe it'll be less of a deal once headers are split :thonk:

Reply:
Yeah I'm not worried about working out the context since they're so different, I guess I'll just avoid the term `sampleState` for anything in audio

Reply:
I really wouldn't worry about confusing anything with SampleState either